styleguide/source/_patterns/01-molecules/09-hub-components/02-hub-card/hub-card-with-select.twig

```diff
@@ -0,0 +1,5 @@
+<div class="hub-card hub-card-with-select" href="#">
+  {% include 'atoms-landscape-3x2' with { 'class': 'hub-card_image' } %}
+  {% include 'atoms-paragraph' with { 'class': 'hub-card_desc', 'content' : 'View the results of the election—the new governing council is ready to serve the Women Physician Section. View the results of the election—the new governing council is ready to serve the Women Physician Section.' } %}
+  {% include 'atoms-select-menu' %}
```

Comment:
As per my comment above, the select menu on this page should load with the following options:
- Select an option
- iTunes|https://www.apple.com/itunes/download/
- Google Play|https://play.google.com/store?hl=en
- Soundcloud|https://soundcloud.com/